scanpipe/models.py

```diff
@@ -561,6 +561,11 @@ class Project(UUIDPKModel, ExtraDataFieldMixin, UpdateMixin, models.Model):
     notes = models.TextField(blank=True)
     settings = models.JSONField(default=dict, blank=True)
     labels = TaggableManager(through=UUIDTaggedItem)
+    project_purl = models.CharField(
+        max_length=2048,
+        blank=True,
+        help_text=_("Project Package URL."),
```

Comment:
This is not helpful, we should instead document the purpose of this field and how it differs from  the usual "Package URL".